smart_card_connector_app/src/window-about-showing.js

```diff
@@ -46,7 +46,7 @@ const openAboutElement =
 function openAboutClickListener(e) {
   e.preventDefault();
 
-  chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  chrome.windows.create({url: ABOUT_WINDOW_URL, type: "popup", width: 700, height: 500});
```

Comment:
nit: This line is longer than 80 characters, which is not allowed by the style guide - please reformat it (you can just run the "format-code.sh" script I added recently).